### PR TITLE
jkashdska

### PR DESCRIPTION
**Pull request by LankyBox01**
ye

### CODE REVIEW

**Comment by RedGuy12** [comment]
you could have made a detailed title and desc...

plus this addon is not very helpful, or following guidelines for addons

**Comment by LankyBox01** [comment]
there are guidelines!?

**Comment by LankyBox01** [comment]
uhhh

**Comment by mxmou** [comment]
There are multiple syntax errors in your code.

**Comment by RedGuy12** [comment]
> there are guidelines!?

i mean, this shouldn't be a persistent script, it's not a module, there's a json syntax error, this isn't a very useful addon, etc, etc, etc

**Comment by LankyBox01** [comment]
Uh how do i fix it!?

**Comment by mxmou** [comment]
> Uh how do i fix it!?

Add a comma before the added line in `addons.json` and on the end of line 9 of `addon.json`, and a slash before the asterisk in the match pattern.

I don't think this pull request will be accepted though, even if you fix it, because this isn't a very useful addon.

File: addons/addons.json [hunk 1]
```diff
@@ -74,4 +74,5 @@
   "show-bbcode",
   "copy-message-link",
   "load-extensions"
```
Comment (RedGuy12) [inline addons/addons.json:76]: add a comma at the end of this line

**Comment by mxmou** [comment]
And please add a meaningful title.

File: addons/rename-forums/addon.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Rename Discussion Forums",
+  "description": "Change &quot;Discuss Scratch&quot; into &quot;Scratch Discussion Forums&quot;.",
+  "userscripts": [
+    {
+      "url": "background.js",
+      "matches": ["https://scratch.mit.edu/discuss*"]
+    }
+  ]
```
Comment (RedGuy12) [inline addons/rename-forums/addon.json:9]: add a comma at the end of this line

File: addons/rename-forums/background.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+if (window.location.href.indexOf("/discuss") > -1) {
+    document.querySelector('.box-head').children[0].innerHTML = 'Scratch Discussion Forums'
+}
```
Comment (RedGuy12) [inline addons/rename-forums/background.js:3]: Put another `}` at the end of your code.

File: addons/rename-forums/background.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+if (window.location.href.indexOf("/discuss") > -1) {
```
Comment (RedGuy12) [inline addons/rename-forums/background.js:1]: put
```
export default async function() {
```
before the first line

File: addons/rename-forums/background.js (hunk 3 above)
Comment (RedGuy12) [inline addons/rename-forums/background.js:3]: rename this file to `userscript.js`

**Comment by LankyBox01** [comment]
I'll do it all over again.

File: addons/rename-forums/addon.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Rename Discussion Forums",
+  "description": "Change &quot;Discuss Scratch&quot; into &quot;Scratch Discussion Forums&quot;.",
+  "userscripts": [
+    {
+      "url": "background.js",
```
Comment (RedGuy12) [inline addons/rename-forums/addon.json:6]: change this to `      "url": "userscript.js",`

**Review by RedGuy12 — CHANGES_REQUESTED** [review]
but this isn't a useful addon anyway

**Pull request closed without merging**